script/update-book2.rb

```diff
@@ -154,6 +154,15 @@ def genbook(language_code, &get_content)
         end
       end
 
+      xlink = html.scan(/href="(https:\/\/git-scm\.com\/.*?[^\/])\#(.*?)"/)
+      xlink&.each do |link|
+        begin
+          html.gsub!("href=\"#{link.first}\##{link[1]}\"", "href=\"#{link.first}/\##{link[1]}\"")
```

Comment:
Maybe these should be converted to relative links in the process, instead of making them absolute again.
Also these are technically broken with this change since the absolute links still point to the rails site.